Week-13/weather/utils/errorResponse.js

```diff
@@ -0,0 +1,8 @@
+class ErrorResponse extends Error {
+  constructor(message, statusCode) {
```

Comment:
Class based component? Can we have a functional component instead?

Reply:
I think I can't, as Error is also class, and I just created custom Error by extending it.